lib/diagnostic.rb

```diff
@@ -69,18 +69,20 @@
 
 # What value will be returned. Write your response as a Ruby string.
 
-Diagnostic.flow_control = 'your response here'
+Diagnostic.flow_control = 'The Dark Night'
 
 # In a Ruby string, write what keyword you use for "else if" clauses in Ruby.
 
-Diagnostic.else_if = 'your response here'
+Diagnostic.else_if = 'elsif'
 
 # Returns in Ruby
 #
 # Does ruby require an explicit return from methods? Answer true or false, and
 # replace nil.
 
-Diagnostic.returns = nil
+Diagnostic.returns = The answer is no it does not require an explicit return,
+you can do it if you would like but not neccessary in Ruby. True or False is
+not the right words to answer this question in my opinion.
 
```

Comment:
`false`

Your answer is correct, but the question specified a boolean to to be returned.